src/ui/server/routes/insurance/your-buyer-details/index.ts

```diff
@@ -0,0 +1,10 @@
+import express from 'express';
+import { YOUR_BUYER } from '../../../constants/routes/insurance/your-buyer';
+import { get as getYourBuyerOrganisation, post as postYourBuyerOrganisation } from '../../../controllers/insurance/your-buyer/your-buyer-details';
```

Comment:
Shoudl it be called `getYourBuyerDetails` instead of `getYourBuyerOrganisation`?